packages/manager/modules/support/src/tickets/ticket/ticket.module.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+
+import ovhUxManagerCore from '@ovh-ux/manager-core';
+import ovhUxNgTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouterAngularJs from '@uirouter/angularjs';
+import angularTranslate from 'angular-translate';
+import 'ovh-ui-angular';
+
+import 'ovh-ui-kit/dist/oui.css';
+import 'ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css';
```

Comment:
Why is it needed, it's all ui-kit components and styles that has been used, right ?